insta/src/App.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import dummyData from './dummy-data';
+import './App.css';
+import PostContainer from './components/PostContainer';
+import SearchBar from './components/SearchBar';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { faComment } from '@fortawesome/free-solid-svg-icons';
+import { faCompass } from '@fortawesome/free-solid-svg-icons';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+library.add(faHeart);
+library.add(faComment);
+library.add(faCompass);
+library.add(faUser);
+library.add(faSearch);
+
+
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      comments: [],
+      commentInput: '',
+      searchInput: '',
+      likes: []
+    };
+  }
+
+  componentDidMount() {
+    const comments = [];
+    const likes = [];
+
+    dummyData.map(data => {
+      return [comments.push(data.comments), likes.push(data.likes)];  
+    })
+
+    this.setState({
+      data: dummyData,
+      comments: comments,
+      likes: likes
+    });
+  }
+
+  addNewComment = (index, event) => {
+    if(event.which === 13) {
+
+      const comment = {
+        username: 'Cyanide6033',
+        text: event.target.value,
+        id: Date.now()
```

Comment:
Two birds with one stone here because we may want to display this data on comments, and it can serve as a key later on 👍